MakeitMVP/CoffeeChat
# Ccl2 111 fe connections

**Summary**
This PR updates components and styling related to the `/connections` route, per our Hi-Fi Figma Design, which includes:
- `<ConnectionsPage />`
- `<Connecting />`
- `<ConnectionOption />`

**Updates**
- Added/updated various images
- Added test data file `src/data/connectionsPageData.jsx`, which is used in `<ConnectionOption />`
- Various layout and styling updates to match Figma design as closely as possible
- `Chat` button now goes to the `/chat/` route

https://github.com/makeitMVPadmin/CoffeeChat/assets/100455148/b1ad5e13-394d-46c9-9c50-512e5b41d17e

**Testing Instructions**
- Pull down this branch locally and run the app (mobile view)
- Click `Connect` in the lower navbar
  - Verify looks good and (decently) matches design
- Click `Connect Me` button
    - Verify looks good and (decently) matches design
- `Chat` and `Meet` buttons should navigate to `/chat` and `/scheduling/:id` routes

**Notes**
- `Verified` icon and `Reviews` tab are not included in this PR
- Could use a bit more refinement and a11y updates
